ballerina/subscriber_client.bal

```diff
@@ -62,37 +62,27 @@ public isolated client class SubscriptionClient {
     isolated remote function unsubscribe(SubscriptionChangeRequest unsubscriptionRequest)
             returns SubscriptionChangeResponse|SubscriptionInitiationError {
         http:Client httpClient = self.httpClient;
-        http:Request builtUnsubscriptionRequest = buildSubscriptionChangeRequest(MODE_UNSUBSCRIBE, unsubscriptionRequest);
-        http:Response|error response = httpClient->post("", builtUnsubscriptionRequest);
+        map<string> request = buildSubscriptionRequest(MODE_UNSUBSCRIBE, unsubscriptionRequest);
```

Comment:
Used `map<string` here because the underlying HTTP request should contain a `x-form-urlencoded` payload. By using `map<string>` and the media-type `x-form-urlencoded` the HTTP client internally does the required conversions.

Reply:
I think when you use map<string>, by default media type is set as `x-form-urlencoded`. Isn't it?
https://github.com/ballerina-platform/ballerina-standard-library/issues/1705#issuecomment-1014781301

Reply:
Fixed with [5f969e536c00bc71513e014548fdc193382d28cb](https://github.com/ballerina-platform/module-ballerina-websub/pull/566/commits/5f969e536c00bc71513e014548fdc193382d28cb)